gunicorn/config.py

```diff
@@ -868,9 +864,13 @@ class ReloadEngine(Setting):
 
         * 'auto'
         * 'poll'
-        * 'inotify' (requires inotify)
+        * 'inotify'
 
         .. versionadded:: 19.7
+
+        .. versionchanged:: 19.8
+           A value of 'inotify' is an alias for 'auto' and uses the
+           best implementation available on the platform.
```

Comment:
Should we also explicitly deprecate `'inotify'` value and remove it in 20?

Reply:
We could, but I'm not sure it's worth the disruption, however minor. It's harmless to me to have it as an alias.

Reply:
You think I should remove the option to pick and always try to use the best? I'm okay with that, too.

Reply:
But I also would like to ship this before R20.